From b4f06abad55916234148ec64d8038322606fc557 Mon Sep 17 00:00:00 2001
From: Jason Lowe <jlowe@apache.org>
Date: Tue, 24 Nov 2015 22:01:03 +0000
Subject: [PATCH 2285/2748] CDH-50592 MAPREDUCE-5883. "Total megabyte-seconds"
 in job counters is slightly misleading.
 Contributed by Nathan Roberts

(cherry picked from commit cab3c7c8892ad33a7eb0955b01e99872ab95e192)

Change-Id: I44b46204402c44bc6e38b37ab4a916f917a211ef
---
 .../apache/hadoop/mapreduce/JobCounter.properties  |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/org/apache/hadoop/mapreduce/JobCounter.properties b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/org/apache/hadoop/mapreduce/JobCounter.properties
index 7a493a8..774002b 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/org/apache/hadoop/mapreduce/JobCounter.properties
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/org/apache/hadoop/mapreduce/JobCounter.properties
@@ -27,9 +27,9 @@ SLOTS_MILLIS_MAPS.name=            Total time spent by all maps in occupied slot
 SLOTS_MILLIS_REDUCES.name=         Total time spent by all reduces in occupied slots (ms)
 MILLIS_MAPS.name=                  Total time spent by all map tasks (ms)
 MILLIS_REDUCES.name=               Total time spent by all reduce tasks (ms)
-MB_MILLIS_MAPS.name=               Total megabyte-seconds taken by all map tasks
-MB_MILLIS_REDUCES.name=            Total megabyte-seconds taken by all reduce tasks
-VCORES_MILLIS_MAPS.name=           Total vcore-seconds taken by all map tasks
-VCORES_MILLIS_REDUCES.name=        Total vcore-seconds taken by all reduce tasks
+MB_MILLIS_MAPS.name=               Total megabyte-milliseconds taken by all map tasks
+MB_MILLIS_REDUCES.name=            Total megabyte-milliseconds taken by all reduce tasks
+VCORES_MILLIS_MAPS.name=           Total vcore-milliseconds taken by all map tasks
+VCORES_MILLIS_REDUCES.name=        Total vcore-milliseconds taken by all reduce tasks
 FALLOW_SLOTS_MILLIS_MAPS.name=     Total time spent by all maps waiting after reserving slots (ms)
 FALLOW_SLOTS_MILLIS_REDUCES.name=  Total time spent by all reduces waiting after reserving slots (ms)
-- 
1.7.9.5

